PrebidMobile/AdUnits/MultiformatAdUnit/PrebidRequest.swift

```diff
@@ -45,6 +47,12 @@ public class PrebidRequest: NSObject {
         super.init()
     }
     
+    // MARK: GPID
+    
+    public func setGPID(_ gpid: String?) {
```

Comment:
We should keep API consistent. Since it is a new API Class, we can consider using only setters for request properties. However, in this case, we should remove other setters like `getExtData` and align this approach with Android.